packages/j-mail/src/handler.ts

```diff
@@ -33,10 +35,21 @@ const getDataForUpdateEmail = async (
   let thanksCount = 0
   let clapCount = 0
 
-  const user = (await prisma.user.findUnique({ where: { id: userId } })) as User
+  const user = (await prisma.user.findUnique({
+    where: {
+      id: userId,
+    },
+  })) as User
```

Comment:
I think you can do this:

```suggestion
  const user = await prisma.user.findUnique({
    where: {
      id: userId,
    },
    include: { configuration: true }
  })
```

and fetch the configuration object at the same time (saving a round trip to the DB)